<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система управления кассовыми аппаратами супермаркета</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>981</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Программа</string>
     </attribute>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>331</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>230</y>
        <width>301</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>230</y>
        <width>301</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="gogogo">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>490</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Запуск</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>520</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>570</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Вызвать техника</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>451</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/kassa.png);background-repeat:no-repeat;background-position:center;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/kassa.png);background-repeat:no-repeat;background-position:center;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>201</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/kassa.png);background-repeat:no-repeat;background-position:center;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Касса №1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Касса №2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>0</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Касса №3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Температура</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Температура</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>200</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Температура</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>476</y>
        <width>961</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="tempKas1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>T =</string>
      </property>
     </widget>
     <widget class="QLabel" name="tempKas2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>T= </string>
      </property>
     </widget>
     <widget class="QLabel" name="tempKas3">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>T =</string>
      </property>
     </widget>
     <widget class="QLabel" name="temp2Kas1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="temp2Kas2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="temp2Kas3">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>150</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Статистика</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>611</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0" rowminimumheight="0,0,0,0,0,0,0,0,0">
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Количество срабатываний датчика температуры термоголовки в кассе 3:  </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Количество срабатываний датчика температуры термоголовки в кассе 1:  </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Количество срабатываний датчика температуры термоголовки в кассе 2:  </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="srTempKas1">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="srTempKas2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="srTempKas3">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>521</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Количество срабатываний датчика аварии ЭКЛЗ в кассе 3:  </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Количество срабатываний датчика аварии ЭКЛЗ в кассе 1:  </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Количество срабатываний датчика аварии ЭКЛЗ в кассе 2:  </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="srEklzKas1">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="srEklzKas2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="srEklzKas3">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>481</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Количество поданных заявок:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Количество срабатываний датчиков температуры всего:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Количество срабатываний датчиков аварии ЭКЛЗ всего:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="kolPod">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="srTempAll">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="srEklzAll">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
